Common/src/main/java/com/hypherionmc/sdlink/util/PKUtil.java

```diff
@@ -0,0 +1,17 @@
+package com.hypherionmc.sdlink.util;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class PKUtil {
+    public static boolean isPK(String id) {
+        try {
+            HttpURLConnection con = (HttpURLConnection) new URL("https://api.pluralkit.me/v2/messages/" + id).openConnection();
+            con.setRequestMethod("GET");
+            return con.getResponseCode() == 200;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
```suggestion
            BotController.INSTANCE.getLogger().error("An error occured while checking message in PluralKit api", e);
            return false;
```
Errors can probably be handled more gracefully here. (suggestion requires importing `com.hypherionmc.sdlink.core.discord.BotController`)